code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -78,10 +78,41 @@
 	speech_args[SPEECH_MESSAGE] = trim(message)
 
 /obj/item/clothing/mask/joy
-	name = "joy mask"
-	desc = "Express your happiness or hide your sorrows with this laughing face with crying tears of joy cutout."
+	name = "Emotional Mask"
+	desc = "Express your happiness or hide your sorrows with this modular cutout."
 	icon_state = "joy"
+	clothing_flags = ALLOWINTERNALS
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
```

Comment:
armored now?

Reply:
theres probably a less shitcode way of making it burn and acid resistant